recipe-server/client/control_new/components/recipes/HistoryItem.js

```diff
@@ -0,0 +1,256 @@
+import { Popover, Alert, Icon, Tag, Timeline } from 'antd';
+import autobind from 'autobind-decorator';
+import { Map } from 'immutable';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'redux-little-router';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+import {
+  REVISION_APPROVED,
+  REVISION_DISABLED,
+  REVISION_LIVE,
+  REVISION_PENDING_APPROVAL,
+  REVISION_REJECTED,
+} from 'control_new/state/constants';
+
+import {
+  getRevisionStatus,
+  isLatestRevision as isLatestRevisionSelector,
+} from 'control_new/state/app/revisions/selectors';
+
+@connect(
+  (state, { revision }) => ({
+    status: getRevisionStatus(state, revision.get('id')),
+    isLatestRevision: id => isLatestRevisionSelector(state, id),
+  }),
+)
+@autobind
+export default class HistoryItem extends React.PureComponent {
+  static propTypes = {
+    isLatestRevision: PropTypes.func.isRequired,
+    revision: PropTypes.instanceOf(Map).isRequired,
+    status: PropTypes.instanceOf(Map).isRequired,
+    selectedRevisionId: PropTypes.string.isRequired,
+    recipeId: PropTypes.string.isRequired,
+    revisionNo: PropTypes.number.isRequired,
+  };
+
+  static StatusStyles = {
```

Comment:
shouldn't this be `statusStyles` or `STATUS_STYLES`? The capitalization of the first `S` suggests this is a class, which it isn't.